src/transactions/test/RevokeSponsorshipTests.cpp

```diff
@@ -579,6 +580,90 @@ TEST_CASE("update sponsorship", "[tx][sponsorship]")
                     checkSponsorship(ltx, a2, 0, nullptr, 0, 2, 1, 0);
                     ltx.commit();
                 }
+
+                SECTION("data")
+                {
+                    DataValue value;
+                    std::string dataName = "test";
+                    auto a1 = root.create("a1", minBal(1));
```

Comment:
Probably better to use `minBal(0)` here, since we should be testing that you can transfer even in the case that the sponsored account has insufficient balance.

Reply:
Fixed